docs/RELEASE_WORKFLOW.md

```diff
@@ -1,36 +1,59 @@
-# Release Workflow
+# How to do releases
+
+## Test releases
+
+* Before every official release we make a test release, similar to "Release
+  Candidates"
+* Test releases are published to supporters and the userbase through our
+  community channels, but not throught he official app stores
+* Create a forum post to inform about the test release, mention what this
+  release is about and what should be tested, here is an
+  [example](https://support.delta.chat/t/help-testing-the-upcoming-1-41-x-release/2793)
+* The goal is to do more broad user testing, looking for issues which came up
+  and gathering feedback
+* Usually after a round of feedback and bug reports we do another test release
+  and repeat, until the release stabilized
+* Optional: We can use flatpack for test releases as well (they have a
+  "testing" branch users can opt-in)
+* Test releases follow the steps described below. Official "stable" releases
+  follow them as well, but require further steps for pushing the builds into
+  the app store
 
 ## Before Releasing
 
-1. make sure no `blocker` bugs are in the issue tracker. If they are some, try to solve them first.
-
-2. Pull translations: `npm run translations-update`
-
-3. Update the local help files. (TODO: information on how do that)
+1. Make sure no `blocker` bugs are in the issue tracker. If they are some, try
+   to solve them first.
+2. Pull translations via `npm run translations-update`.
+3. Update the local help files.
```

Comment:
optional: draft a device message for the new version informing users of the release highlights and also thanking the testers for testing the app (only for test releases)

for the real release the device message should exist, if there is no highlight to mention we can say it's a realease focused on stability and bug fixing